guides/next-version.md

```diff
@@ -0,0 +1,19 @@
+Next Version
+===
```

Comment:
why not just just headings?

```suggestion
# Next Version
```

Reply:
both create a `<h1>`, but sure. i was planning on adding a linter in a future PR anyways